src/EnergyPlus/UnitarySystem.cc

```diff
@@ -3067,6 +3074,27 @@ namespace UnitarySystems {
                 this->m_DesignCoolingCapacity = DataSizing::AutoSize;
                 if (this->m_HeatingCoilType_Num == DataHVACGlobals::Coil_HeatingWaterToAirHP)
                     EqSizing.DesHeatingLoad = state.dataSize->DataConstantUsedForSizing;
+                // airflow sizing with multispeed fan
```

Comment:
I want to make a general comment here. The "scope Hungarian" notation `m_` means "member variable". Prepending access with `this->` is only necessary in cases where a member variable is shadowed (i.e., has the same name) as a local variable, e.g., `this->name = name;` where the first `name` is the member and the second one is the local. Otherwise it is not necessary because C++ assumes that if a variable inside a method is not local then it must be a member. [Note: global variables have lower scope precedence than member variables, but we don't really have these in EnergyPlus anymore except for the various `xNamesUC` arrays].

Anyways, if all member variables start with `m_` and local variables do not, then there is no possibility of name shadowing and therefore no need for `this->`.

TL;DR `this->m_X` is redundant from both a compiler and readability standpoint. The `m_` tells both the compiler and the reader the same thing, i.e., that this variable is a member of the class that defines the current method. I would eliminate one of them, preferably `this->` because it is longer and thus more annoying. Also, it's a keyword so it gets colorized and makes the whole screen of that color.

Reply:
@amirroth I agree with you. The changes you suggested is beyond this new feature scope. It is better to make the changes in the future factory.